tools/travis/run_tests.sh

```diff
@@ -27,6 +27,10 @@ fi
 
 test="$1"
 
+if [ "$#" -gt 1 ]; then
```

Comment:
It can error if greater, or just ignore any additional args. I preferred just to ignore, but I can make it more strict. And I put it in an `if` just because I'm too used to real languages. It's hard for me to keep in mind that `pattern="$2"` would work even when there's no $2. I will make the changes on monday.